content/src/fieldConfig/config.json

```diff
@@ -213,13 +213,18 @@
     "reinstatementPendingPermitStatusText": "Reinstatement Pending",
     "permitStatusText": "Permit Status:",
     "submittedStatusText": "Submitted",
+    "applicationChecklistItems": "Application Checklist Items",
```

Comment:
[sand] Should this end with the word "Text"? Looking at its siblings, this seems to be a pattern.

Reply:
It's such a lengthy key name which is one reason I left it out (I see `reinstatementPendingPermitStatusText`and probably would have left out `text` there had I created that one). Another thought: Essentially these are all text and not every single key ends with text. I am okay with it either way and will make the change.